Config/Project64.rdb

```diff
@@ -5825,8 +5825,8 @@ Self Texture=1
 Good Name=The Legend of Zelda - Ocarina of Time (E) (GC)
 Internal Name=THE LEGEND OF ZELDA
 Status=Compatible
-Cheat0=801D8F8B 0002 //Subscreen Delay Fix
-Cheat1=D109A8E4 0320,8109A8E4 0000,D109A8E6 F809,8109A8E6 0000 //End Credits Fix
+Counter Factor=1 //Actual Subscreen Delay Fix
```

Comment:
setting cf to 1 is not a real fix, it is to do with clearing the background in the gfx plugin, as jabo's does not modify the rdram this does not fix things. So a true fix is in the gfx plugin

Reply:
It does help with Glide64 and GLideN64. CF=1 matches console (subscreen appears right before jingle finishes playing). I verified this with console footage. CF = 2, subscreen appears moments _after_ jingle has played out.

So CF=1 does help with accurate gfx plugins. Do you just want @AriaHiro64 to leave the Cheat0 intact for Jabo (which is not even the default anymore and Project64 Video seems to work fine without the fix). The fix is not even getting applied to Project64 Video because the plugin is not called "Glide64 for Project64" as it is hardcoded in the RDB for that fix. So it only helps Jabo.

Reply:
Jabo is literally the only plugin that's unaffected

Reply:
Jabo has its own bug and shouldn't be a basis on every other plugin

Reply:
I've tested rice angrylion gliden64 and glide and all benefit

Reply:
Yes jabo needs improvement and I'm not trying shit on jabo when I say this but you would be hard-pressed to find a worse option for oot in it's current state